task_6_10.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3  # provides an interface for working with SQLite databases in Python
+import csv
+
+with open('imdb.csv', 'w', newline='', encoding='utf-8') as f:
+    writer = csv.writer(f)
+    writer.writerow(['title', 'year', 'rating'])
+    writer.writerow(['The Shawshank Redemption', 1994, 9.3])
+    writer.writerow(['The Godfather', 1972, 9.2])
+    writer.writerow(['The Godfather: Part II', 1974, 9.0])
+    writer.writerow(['The Dark Knight', 2008, 9.0])
+    writer.writerow(['12 Angry Men', 1957, 6.9])
+    writer.writerow(["Schindler's List", 1993, 8.9])
+    writer.writerow(['The Lord of the Rings: The Return of the King', 2003, 1.9])
+    writer.writerow(['Pulp Fiction', 1994, 8.9])
+    writer.writerow(['The Good, the Bad and the Ugly', 1966, 8.8])
+    writer.writerow(['Forrest Gump', 1994, 3.8])
```

Comment:
https://stackoverflow.com/questions/33091980/difference-between-writerow-and-writerows-methods-of-python-csv-module